### PR TITLE
added multi search

### PR DESCRIPTION
**Pull request by Erryk20**
@Alrok @SergeyKondratovec @SergiiBondarchuk @Viktorminator @sauron918 @comaw @jusnak3 @ddis @yurikzt

### CODE REVIEW

**Comment by jusnak3** [comment]
Use default msearch functionality:
https://discuss.elastic.co/t/elasticsearch-5-0-php-msearch/66716

**Review by ghost — CHANGES_REQUESTED** [review]
Use default msearch functionality:
https://discuss.elastic.co/t/elasticsearch-5-0-php-msearch/66716

**Review by ddis — CHANGES_REQUESTED** [review]
https://discuss.elastic.co/t/elasticsearch-5-0-php-msearch/66716

File: src/Elasticsearch/ClientBuilder.php [hunk 1]
```diff
@@ -0,0 +1,351 @@
...
+/**
+ * Class ClientBuilder
+ *
+ * @category Elasticsearch
+ * @package  Elasticsearch\Common\Exceptions
+ * @author   Zachary Tong <zach@elastic.co>
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache2
+ * @link     http://elastic.co
+ */
+class ClientBuilder extends \Elasticsearch\ClientBuilder
```
Comment (comaw) [inline src/Elasticsearch/ClientBuilder.php:21]: Do not rewrite the entire class. Use the programming patterns of the reassembly of private entities

File: src/Resources/Resource.php [hunk 2]
```diff
@@ -184,6 +184,23 @@ public function filter($filter)
...
+     * Set filters to the resource selecting
+     *
+     * @param array $filters
+     * @return $this
+     */
+    public function filters(array $filters)
+    {
+
+        /** @var MultiFullTextFilter $filter */
+        foreach ($filters AS $filter) {
```
Comment (comaw) [inline src/Resources/Resource.php:197]: $this->filters = array_merge(array_values($filters));

Use instead foreach

File: src/Elasticsearch/Endpoints/Search.php [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+     */
+    public function getURI()
+    {
+        $index = $this->index;
+        $type = $this->type;
+
+        $uri = $this->multi ? "/_msearch" : "/_search";
+
+        if (isset($index) === true && isset($type) === true) {
+            $uri = "/$index/$type".$uri;
```
Comment (comaw) [inline src/Elasticsearch/Endpoints/Search.php:37]: use one standard - or "" or concatenation

**Pull request closed without merging**